io/disk/htx_block_devices.py

```diff
@@ -68,38 +72,86 @@ def setup_htx(self):
         Builds HTX
         """
         packages = ['git', 'gcc', 'make']
-        detected_distro = distro.detect().name
-        if detected_distro in ['centos', 'fedora', 'rhel', 'redhat']:
+        self.detected_distro = distro.detect()
+        if self.detected_distro.name in ['centos', 'fedora', 'rhel', 'redhat']:
             packages.extend(['gcc-c++', 'ncurses-devel', 'tar'])
-        elif detected_distro == "Ubuntu":
+        elif self.detected_distro.name == "Ubuntu":
             packages.extend(['libncurses5', 'g++', 'ncurses-dev',
                              'libncurses-dev', 'tar'])
-        elif detected_distro == 'SuSE':
+        elif self.detected_distro.name == 'SuSE':
```

Comment:
self.detected_distro.name  calling .name 4 times... instead earlier `detected_distro = distro.detect().name`   was called one time.. and the param can be sued in 4 places.. any reason to make it multiple lines ?